docs/docs/.vuepress/config.js

```diff
@@ -58,12 +58,12 @@ module.exports = {
           "/workflows/",
```

Comment:
I removed legacy docs from the sidebar, and I'm not referencing them anywhere in the docs (the docs still exist). To spur the transition to component actions, I thought this might be preferred. Let me know if you want me to reference in a specific place.